lib/internal/url.js

```diff
@@ -701,7 +701,7 @@ class URLSearchParams {
   }
 
   [util.inspect.custom](recurseTimes, ctx) {
-    if (!this || !(this instanceof URLSearchParams)) {
+    if (!this || this[searchParams] === undefined) {
```

Comment:
The `searchParams` symbol is used by the `URL` class as well, so this will no longer error out on `URL` objects.